docs/architecture/_index.md

```diff
@@ -0,0 +1,188 @@
+
+{{< /expand >}}
+
+## Communication
+
+<img src="https://assets-global.website-files.com/62fba5686b6d47fe2a1ed2a6/63334e5e19a68d329b1c5b0e_viam-overview-illustrations-manage.svg" alt="A diagram illustrating secure machine control." class="alignleft" style="max-width:270px;"></img>
+
+Viam uses peer-to-peer communication, where all machines running `viam-server` or the [micro-RDK](/installation/micro-rdk-dev/) (the version of `viam-server` for microcontrollers) communicate directly with each other as well as with the cloud.
+This peer-to-peer connectivity is enabled by sending [gRPC commands over WebRTC connections](/architecture/machine-to-machine-comms/#low-level-inter-robotsdk-communication).
+
+On startup, `viam-server` establishes a {{< glossary_tooltip term_id="webrtc" text="WebRTC" >}} connection with the [Viam app](https://app.viam.com).
+`viam-server` pulls its configuration from the app, caches it locally, and initializes all components and services based on that configuration.
+
+If [sub-parts or remote parts](#complex-machines-with-multiple-parts) are configured, communications are established between the `viam-server` instances on each of them.
+
+If you have client code running on a separate computer, that code sends API requests to `viam-server` using gRPC over WebRTC.
+If a WebRTC connection cannot be established, the request is sent directly over gRPC.
+When a built-in service communicates with a component, for example when the vision service requests an image from a camera, `viam-server` handles that request as well.
+
+When you control your machine or view its camera streams or sensor outputs from the Viam app **CONTROL** tab, those connections happen over WebRTC.
+The Viam app hits the same API endpoints as your SDK client code, with `viam-server` handling requests.
+
+{{% alert title="Protobuf APIs" color="info" %}}
+All Viam APIs are defined with the [Protocol Buffers (protobuf)](https://protobuf.dev/) framework.
+This is what enables Viam to provide SDKs in a variety of different programming languages.
+{{% /alert %}}
+
+For more details, see [Machine-to-Machine Communication](/architecture/machine-to-machine-comms/).
+
+### Security
+
+TLS certificates certificates provided by the Viam app ensure that all communication is authenticated and encrypted.
+
+Viam uses API keys with [role-based access control (RBAC)](/cloud/rbac/) to control access to machines from client code.
+
+## Data management flow
+
+{{<imgproc src="/architecture/data-flow.svg" resize="x1100" declaredimensions=true alt="Data flowing from local disk to cloud to the Viam app, SDKs, and MQL and SQL queries." >}}
+<br>
+
+Data is captured and synced to the Viam cloud as follows:
+
+1. Data collected by your components, such as sensors and cameras, is first stored locally in a specified directory (defaults to <file>~/.viam/capture</file>).
```

Comment:
```suggestion
1. Data collected by your resources, such as sensors and cameras, is first stored locally in a specified directory (defaults to <file>~/.viam/capture</file>).
```
Since it can be for services as well